analyzers/rspec/cs/S2053_c#.json

```diff
@@ -25,6 +25,12 @@
     ],
     "OWASP Top 10 2021": [
       "A2"
+    ],
+    "PCI DSS 3.2": [
+      "6.5.10"
+    ],
+    "PCI DSS 4.0": [
+      "6.2.4"
     ]
```

Comment:
My understanding is that this kind of changes are related to this card: https://trello.com/c/U43VsQCX